client/app/account/account.routes.js

```diff
@@ -4,7 +4,6 @@ angular.module('App').config(($stateProvider) => {
     views: {
       'app@': {
         templateUrl: 'account/index.html',
-        controller: 'AppCtrl',
```

Comment:
I need this in an upcoming PR, please don't remove 😄

Reply:
@FredericEspiau imho it doesn't make sense to use global app controller for a module if behaviour is specific to the module